### PR TITLE
DDCE-5795 -ers-file-validator - Bootstrap-play upgrade

### PR DESCRIPTION
**Pull request by maheshhvr**
DDCE-5795 -ers-file-validator - Bootstrap-play upgrade

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -3,30 +3,30 @@ import sbt.*
...
+    "commons-codec"           % "commons-codec"              % "1.17.1",
     "uk.gov.hmrc.mongo"      %% "hmrc-mongo-play-30"         % mongoVersion,
     "org.apache.pekko"       %% "pekko-connectors-csv"       % pekkoVersion
   )
 
   val test: Seq[ModuleID] = Seq(
     "org.apache.pekko"        %% "pekko-connectors-csv"     % pekkoVersion,
     "org.apache.pekko"        %% "pekko-connectors-xml"     % pekkoVersion,
-    "org.apache.pekko"        %% "pekko-testkit"            % pekkoVersion,
+    "org.apache.pekko"        %% "pekko-testkit"            % "1.0.3",
```
Comment (matthew-gibson-18) [inline project/AppDependencies.scala:24]: Why are we updating just the testkit one and not the val for pekkoVersion on line 7

**Review by matthew-gibson-18 — COMMENTED** [review]
Why are we turning off magic.number in a number of places?

File: project/AppDependencies.scala (hunk 1 above)
Comment (maheshhvr) [reply project/AppDependencies.scala:24]: pekkoVersion is also used for pekko-connectors-csv  in line 22. Pekko-connectors-csv is not compatible with pekkoVersion 1.0.3 (latest version for pekko-connectors-csv is 1.1.0 and previous version 1.0.2)

**Comment by maheshhvr** [comment]
> Why are we turning off magic.number in a number of places?

There were a lot of warnings related to magic number.Turned off the magic.number to remove the warnings and had to do it in number of places to remove all the warnings

**Comment by maheshhvr** [comment]
> * File [conf/application-json-logger.xml](https://github.com/hmrc/ers-file-validator/blob/DDCE-5795/conf/application-json-logger.xml#L1) can be removed. Logging is provided by Bootstrap (since `8.6.0`) and can be configured via application config alone - See confluence https://confluence.tools.tax.service.gov.uk/x/9oH1FQ

-File has been removed as suggested

**Review by matthew-gibson-18 — APPROVED** [review]

**Pull request merged**